cfe_internal/enterprise/federation/federation.cf

```diff
@@ -489,7 +489,7 @@ bundle agent setup_status
     "ssh_server_fingerprint"
       # ssh-keyscan is used because it's more reliable/easy than trying to
       # parse sshd config to find the file and then readfile():
-      string => execresult("ssh-keyscan -t ecdsa localhost 2>/dev/null | sed s/localhost//g", useshell);
+      string => execresult("ssh-keyscan -t ecdsa localhost 2>/dev/null | sed 's/localhost //g'", useshell);
```

Comment:
Wait, didnt we change from ssh host key to cfengine id?

Reply:
This is the fingerprint so should be fine.

Reply:
Oh, okay